hmrc-mongo-test-play-26/src/main/scala/uk/gov/hmrc/mongo/test/PlayMongoCollectionSupport.scala

```diff
@@ -0,0 +1,29 @@
+/*
+ * Copyright 2020 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package uk.gov.hmrc.mongo.test
+
+import org.mongodb.scala.bson.BsonDocument
+import org.mongodb.scala.model.IndexModel
+import uk.gov.hmrc.mongo.play.json.PlayMongoRepository
+
+trait PlayMongoRepositorySupport[A] extends MongoCollectionSupport {
```

Comment:
Perhaps this is a bit early, seeing as how we haven't written user-documentation yet... but should we add some scaladoc about why this trait exists, and how people are expected to use it?